mysql/errcode.go

```diff
@@ -881,7 +881,8 @@ const (
 	ErrAlterOperationNotSupportedReasonNotNull                      = 1861
 	ErrMustChangePasswordLogin                                      = 1862
 	ErrRowInWrongPartition                                          = 1863
-	ErrErrorLast                                                    = 1863
+	ErrUnknownStmtFetchResultSet                                    = 1864
```

Comment:
```
Error: 1864 SQLSTATE: HY000 (ER_MTS_EVENT_BIGGER_PENDING_JOBS_SIZE_MAX)

Message: Cannot schedule event %s, relay-log name %s, position %s to Worker thread because its size %lu exceeds %lu of slave_pending_jobs_size_max.
```
errcode 1864 is not the code return by MySQL, MySQL will return `1243` in this case

```
#define ER_UNKNOWN_STMT_HANDLER 1243

  if (!(stmt= thd->stmt_map.find(stmt_id)))
  {
    char llbuf[22];
    my_error(ER_UNKNOWN_STMT_HANDLER, MYF(0), static_cast<int>(sizeof(llbuf)),
             llstr(stmt_id, llbuf), "mysqld_stmt_fetch");
    DBUG_VOID_RETURN;
  }
```
You can find the error info in https://dev.mysql.com/doc/refman/5.7/en/error-messages-server.html